apps/trading/pages/client-router.tsx

```diff
@@ -40,11 +53,15 @@ const routerConfig = [
     element: <LazyMarkets />,
   },
   {
-    path: 'markets/:marketId',
+    path: Routes.SKELETON_MARKET,
```

Comment:
suggestion (non blocking): I feel like it might be worth exploring a nested route here as /markets/abc is essentially a child of /markets

This way we might not need the skeleton prop, instead relying on the if there is a route match instead

Reply:
I've removed the prop as it was not really needed - also changed to the nested routes as suggested.